build/Linux+BSD/jessie-packaging-armhf.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ericfont/armv7hf-debian-qemu:jessie
+
+RUN [ "cross-build-start" ]
+
+RUN apt-get update
+
+# need to be able to use https for wget
+RUN apt-get install ca-certificates wget #git
+
+# get prebuilt AppImageKit
+RUN wget https://bintray.com/artifact/download/ericfont/prebuilt-AppImageKit/AppImageKit-5_built-in-armv7hf-jessie.tar.gz
```

Comment:
note: currently there are no official AppImageKit binaries for armhf based off their v5 release code, so I built my own binary which I wget from bintray here.  Once AppImageKit provides and official binary, would probably edit this line to wget from their official binary release.